src/generate/rust.rs

```diff
@@ -511,44 +444,60 @@ impl<Pat> Continuation<'_, Pat> {
 
     fn reify_as(&mut self, label: Rc<CodeLabel>) {
         let code = self.to_inline();
-        let code = quote!(#label => {#code});
+        let label_src = label.to_src();
+        let code = quote!(#label_src => {#code});
         self.code_label_arms.push(code);
         self.code = Code::Label(label);
     }
 }
 
 trait ContFn<Pat> {
-    fn apply(self, cont: Continuation<'_, Pat>) -> Continuation<'_, Pat>;
+    fn apply<'a, 'r>(self, cont: Continuation<'a, 'r, Pat>) -> Continuation<'a, 'r, Pat>;
     // HACK(eddyb) `Box<dyn FnOnce<A>>: FnOnce<A>` is not stable yet,
```

Comment:
[This is stable now.](https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=fd1a7f1591a6bfa11fe730673e41fb01)